### PR TITLE
Correct difference crop between Python2 and Python3

### PR DESCRIPTION
**Pull request by maiagripp**
## Motive
After the migration from Thumbor 6.0 to Thumbor 7.0, we were looking for the expected response were same as actual response, but instead we get the crop a little bit different.

## Changes Proposed
Remove the forced cast to ```int``` and adding a customize ```round``` method which maintain the same behaviour as ```round``` in Python2.

## How to validate this PR
```
cd thumbor/

docker build --platform linux/amd64 -f TestDockerfile39 -t thumbor-test-39 .

docker run --rm -it --name thumbor-test --memory 512mb -p 8888:8888 --platform linux/amd64 -v "$(pwd):/app" thumbor-test-39 make setup run

docker exec -it thumbor-test bash

make unit
```

Closes https://github.com/thumbor/thumbor/issues/1472

### CODE REVIEW

**Comment by mvhirsch** [comment]
Thank you for contributing @maiagripp!

@heynemann this looks well tested and sounds reasonable to me. Could you please review these changes?

File: thumbor/point.py [hunk 1]
```diff
@@ -49,19 +49,19 @@ def __init__(
...
         self.origin = origin
 
     @classmethod
     def from_square(
         cls, x, y, width, height, origin="detection"
     ):  # pylint: disable=invalid-name
-        center_x = x + width // 2
-        center_y = y + height // 2
+        center_x = x + (width // 2)
+        center_y = y + (height // 2)
```
Comment (scorphus) [inline thumbor/point.py:64]:
```suggestion
        center_x = x + width // 2
        center_y = y + height // 2
```

**Review by scorphus — COMMENTED** [review]
I still need to dig into this one myself to better understand it. From a first look it seems it needs a more solid fix, if it is indeed an issue.

File: thumbor/point.py (hunk 1 above)
Comment (maiagripp) [reply thumbor/point.py]: hey @scorphus !
thanks for the suggestion. It was already implemented.

**Comment by maiagripp** [comment]
@scorphus ping🙂

**Review by scorphus — COMMENTED** [review]
I don't think there is an issue here. I think The image used here is misleading. It being so small and one pixel wider than its height — 37x36 — makes this seem like an issue.

Consider the following samples and their “croppings” by both Thumbor 6 and 7:

## Sample image of size 36x36

Both cropped images are identical. That's because the width, 36, is an even number.

### Original image

thumbor-issue-1475-36x36

### Cropped by Thumbor 6

thumbor-issue-1475-36x36-thumbor-6

### Cropped by Thumbor 7

thumbor-issue-1475-36x36-thumbor-7

## Sample image of size 37x36

The cropped images are different, as 37 is odd. Thumbor 6 crops it one pixel less from the left, Thumbor 7 from the right.

### Original image

thumbor-issue-1475-37x36

### Cropped by Thumbor 6

thumbor-issue-1475-37x36-thumbor-6

### Cropped by Thumbor 7

thumbor-issue-1475-37x36-thumbor-7

## Sample image of size 38x36

Again, both cropped images are identical.

### Original image

thumbor-issue-1475-38x36

### Cropped by Thumbor 6

thumbor-issue-1475-38x36-thumbor-6

### Cropped by Thumbor 7

thumbor-issue-1475-38x36-thumbor-7

Some could argue that Thumbor 6 and 7 should behave the same. I argue Thumbor 6 had it wrong. The image should be cropped from the right.

**Comment by heynemann** [comment]
Amazing investigation Pablo!!! Thanks for that and 100% agree!!!

**Comment by heynemann** [comment]
Closing this as the fix does not seem to be correct. If you still feel we need to pursue this re-open with the reasoning.

**Pull request closed without merging**